notebooks-auth/resources/user_test_cowbird_jupyter.ipynb

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
<p>Add the prefix <code>notebook-auth/test_cowbird_jupyter.ipynb</code> to make the reference more obvious.</p><p> </p><p>Add a markdown link for <code>jhub-client</code> to have the specific utility reference.</p><p> </p><p>Seems like some syntax error in "This required the code found" -&gt; "This requires that the code found... " ?</p><p> </p><p>I think it would be a good idea to indicate the security issue related to mounting the socket, which is a strong point against using that approach, and why <code>jhub-client</code> was preferable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>